redis_test.go

```diff
@@ -24,7 +27,12 @@ const (
 	aclCat          = "+@admin"
 )
 
+var redisTls = false
+
 func prepareRedisTestContainer(t *testing.T) (func(), string, int) {
+	if os.Getenv("REDIS_TLS") != "" {
```

Comment:
Hmm, I'm wondering how the redis container in this test is configured for TLS 🤔 Do you know if the TLS tests are passing currently? I would expect the redis container to be configured with a certificate and private key for TLS to work.

Reply:
Ah, I'm sorry. I forgot that we're supplying our own running redis container for the TLS test. Disregard this.